services/movie_session.py

```diff
@@ -0,0 +1,65 @@
+from django.db.models import QuerySet
+
+from db.models import MovieSession
+import datetime as dt
+
+
+def create_movie_session(
+        movie_show_time: dt.datetime,
+        movie_id: int,
+        cinema_hall_id: int
+) -> MovieSession:
+    movie_session = MovieSession.objects.create(
+        show_time=movie_show_time,
+        movie_id=movie_id,
+        cinema_hall_id=cinema_hall_id
+    )
+    return movie_session
+
+
+def get_movies_sessions(
+        session_date: dt.datetime = None
+) -> QuerySet:
+    movie_session = MovieSession.objects.all()
+    if session_date:
+        movie_session = MovieSession.objects.filter(
```

Comment:
No need to write again MoviSession.objects. better to do as 
```suggestion
        movie_session = movie_session.filter(
```